docs/quick_start.rst

```diff
@@ -145,7 +145,7 @@ Now that there are some packages in the workspace, Catkin has something to build
 .. note::
 
     Catkin utilizes an "out-of-source" and "aggregated" build pattern. This
-    means that not only will temporary or final build products never be placed
+    means that not only will temporary or final build products ever be placed
```

Comment:
The text you posted there sounds right to me. If you'll update the pr and squish the commits I'll merge in the morning. Thanks again!